src/identity/components/Name.tsx

```diff
@@ -23,8 +24,10 @@ export function Name({
     );
   }
 
+  const resolvedAddress = contextAddress ?? address;
```

Comment:
why the word `resolved`? I think is best just to call it `accountAddress`.

Reply:
Because that's the address that we settle on between `contextAddress` and `address`. 

I'll update to `accountAddress`